scripts/helpers.js

```diff
@@ -4,11 +4,9 @@
  */
 
 const shell = require('shelljs');
-const fs = require('fs');
 
 const exerciseDirs = shell.ls('-d', 'exercises/*');
 
-const config = JSON.parse(fs.readFileSync('config.json'))['exercises'];
```

Comment:
eslint told me these variables were unused.

Reply:
I think we used it and then you made the scripts better ;)